### ISSUE TITLE
Issues fetching source files.

### ISSUE THREAD

**Issue by dianakhuang** [transifex/cli#24]
I am in the process of migrating a project between two organization, and I'm hitting this error when pulling down source files.

I am able to pull down translations (using `tx pull -a`), and that works great, but when I try to pull down the the source files with `tx pull -s`, I get the error:

```
 Resource o:open-edx:p:edx-themes:r:themespo
 SUCCESS  Organization 'open-edx' fetched
 SUCCESS  Project 'edx-themes' fetched
Downloading source files
 SUCCESS  Resource themespo fetched
  ERROR   Downloading source file themes.po: directory '' does not exist
directory '' does not exist
```

Running with cli version 1.0.2 on macOS.

**Comment by codegaze** [comment]
Hello @dianakhuang, thanks for bringing this to our attention!

We just released a new version `v1.0.3` that should fix an assumption we made and resulted to this issue. 
Please feel free to try it out and close the issue! Cheers!

**Comment by dianakhuang** [comment]
@codegaze Thank you so much for your prompt response! It works great now.
